Game_Manual/Levels_and_Roles.md

```diff
@@ -0,0 +1,156 @@
+# Levels and Roles
+
+Players advance through various levels. Each level has specific requirements and confers new abilities to the player.
+
+Roles delineate different ways of interacting with the game. Players at all levels have the "Player" role, and as players advance they gain access to new roles.
+
+## Levels
+
+Levels are cumulative, so more advanced levels also include privileges of previous levels.
+
+Every level is a voting pool. In other words, players vote on goals that are in their level with other players in their level.
```

Comment:
This makes it sound like I can't vote for goals below my level, which was not my understanding. Might need to clarify this.